### DISCUSSION TITLE
Post reading time

### DISCUSSION THREAD

**Discussion by paddy-exe** [hugo-sid/hugo-blog-awesome#12] [Feature requests]
A nice view of the estimated reading time to display for the user (also from Papermod theme)
![image](https://user-images.githubusercontent.com/38077837/225322693-d83d3bab-83d0-4418-b557-c76de3a5a9a0.jpeg)

**Comment by mattcen** [comment]
Ref: `ReadingTime` on https://gohugo.io/variables/page/#page-variables
